index.html

```diff
@@ -2,12 +2,75 @@
 <html>
 <head>
 <link href="style.css" rel="stylesheet" type="text/css">
+<script src="https://kit.fontawesome.com/173db1d6d6.js" crossorigin="anonymous"></script>
 <title>Portafolio</title>
 </head>
 <body>
 
-<h1>This is a Heading</h1>
-<p>This is a paragraph.</p>
+    
+<div class="wrapper">
+
+    <nav class="toolbar">
+        <ul>
+            <li class="item">Edahí Gutiérrez </li>        
+            <li id="item-right" class="item">&#9776;</li>                    
+        </ul>        
```

Comment:
- For better accessibility reasons, please wrap your logo/name in an `a` tag that navigates back to the home page when clicked. This would make for a better user experience.
- I would not advise that you wrap up your logo/name and menu icon in a `ul`, you can use a `div` instead. This would make for a better HTML semantic 🙏

Reply:
Thanks. I agree